homeassistant/components/open_meteo/__init__.py

```diff
@@ -0,0 +1,57 @@
+"""Support for Open-Meteo."""
+from __future__ import annotations
+
+from open_meteo import Forecast, OpenMeteo, OpenMeteoError
+
+from homeassistant.components.weather import DOMAIN as WEATHER_DOMAIN
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import ATTR_LATITUDE, ATTR_LONGITUDE, CONF_ZONE
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator, UpdateFailed
+
+from .const import DOMAIN, LOGGER, SCAN_INTERVAL
+
+PLATFORMS = (WEATHER_DOMAIN,)
```

Comment:
Use the helper 😝

Reply:
![](https://media3.giphy.com/media/QUENDfi6DEMLzQ0CKt/giphy.gif)

Reply:
Watching this comment... what helper?

Reply:
<https://github.com/home-assistant/core/pull/60857/files#diff-07e20eb7c6560330f9774eb23343f5dd3002106bb8b6523ccb18d95857990d77R22>

Reply:
Done